### PR TITLE
inv: implementation for all subtypes of AbstractCliffordOperator

### LINKED ISSUE #308: `inv` implementation for all subtypes of `AbstractCliffordOperator` [$200]
<details>
<summary><strong>Bug bounty logistic details</strong> (click to expand)</summary>

To claim exclusive time to work on this bounty either post a comment here or message [skrastanov@umass.edu](mailto:skrastanov@umass.edu) with:

- your name
- github username
- **(optional)** a brief list of previous pertinent projects you have engaged in

Currently the project is claimed by `no one` until `...`.

If you want to, you can work on this project without making a claim, however claims are encouraged to give you and other contributors peace of mind. Whoever has made a claim takes precedence when solutions are considered.

You can always propose your own funded project, if you would like to contribute something of value that is not yet covered by an official bounty.
</details>

# Project: `inv` implementation for all subtypes of `AbstractCliffordOperator` [$200]

`LinearAlgebra.inv` already has methods defined for some of the types in this library meant to represent a Clifford operation. To obtain this bounty, you need to implement the missing `inv` methods for all other subtypes of `AbstractCliffordOperator`.

**Required skills**: Understanding of the Stabilizer state formalism.

**Reviewer**: Stefan Krastanov

**Duration**: 1 month

#### Payout procedure:

The Funding for these bounties comes from the National Science Foundation and from the NSF Center for Quantum Networks. The payouts are managed by the NumFOCUS foundation and processed in bulk once every two months. If you live in a country in which NumFOCUS can make payments, you can participate in this bounty program.

[Click here for more details about the bug bounty program.](https://github.com/QuantumSavory/.github/blob/main/BUG_BOUNTIES.md)

### PR DESCRIPTION
**Pull request by thofma**
Let me know if I missed something.

Should close #308.

### CODE REVIEW

File: test/test_symcliff.jl [hunk 1]
```diff
@@ -78,3 +78,13 @@
         end
     end
 end
+
+@testset "TwoQubitOperator inv methods" begin
+    gate_types = [sCNOT, sCPHASE, sSWAP, sXCX, sXCY, sXCZ, sYCX, sYCY, sYCZ, sZCX, sZCY, sZCZ, sZCrY, sZCrYinv]
```
Comment (Krastanov) [inline test/test_symcliff.jl:83]: Would this work with `subtypes(AbstractTwoQubitOperator)`? That way nothing would be missed in the future if other gates get added.

**Comment by Krastanov** [comment]
FYI, some of this work has been merged thanks to another contributor, so I will need to split the bounty on this. https://github.com/QuantumSavory/QuantumClifford.jl/pull/315

**Comment by thofma** [comment]
Hm, my bad. I did not see any PR linked in original issue. I am happy to let @Fe-r-oz finish this. No need to split. He can copy as much as he wants from here.

**Comment by Krastanov** [comment]
Thanks, Tommy, this is very much appreciated!

**Comment by Fe-r-oz** [comment]
Thank you, Tommy, It's greatly appreciated. It should have been me who should have closed my PR. Yours look more polished. 

It would be bad if bounty is not split given your introduction to new fixes!

**Comment by Fe-r-oz** [comment]
@krastanov, I completely agree on splitting the bounty as technically I was not assigned to it!

**Comment by Krastanov** [comment]
@Fe-r-oz , I will message Tommy directly and take care of it. Please feel free to finish your PR and reuse what he has done here.

**Pull request closed without merging**